src/main/java/org/opensrp/validator/InventoryDataValidator.java

```diff
@@ -62,19 +74,18 @@ public InventoryDataValidator(List<String> validationErrors) {
 		validDonors = getValidDonors();
 		validUnicefSections = getValidUnicefSections();
 
-		Date deliveryDate = deliveryDateInString != null ? convertStringToDate(deliveryDateInString) : null;
 		ProductCatalogue productCatalogue;
 		PhysicalLocation physicalLocation;
 		productCatalogue = productCatalogId != null ?
 				productCatalogueService.getProductCatalogue(Long.valueOf(productCatalogId), "") :
 				null;
-		physicalLocation = locationId != null ? physicalLocationService.getLocation(locationId, true, false) : null;
+		physicalLocation = locationId != null ? physicalLocationService.getStructure(locationId, true) : null;
```

Comment:
why are we changing to structures

Reply:
Since, as per the flow of add inventory, the service point Id should be the structure Id.
Please refer to this issue https://github.com/OpenSRP/opensrp-server-web/issues/650 @githengi